src/ngx_http_lua_ssl_certby.c

```diff
@@ -854,6 +854,47 @@ ngx_http_lua_ffi_ssl_server_name(ngx_http_request_t *r, char **name,
 }
 
 
+int
+ngx_http_lua_ffi_ssl_server_port(ngx_http_request_t *r, unsigned int *port,
+    char **err)
+{
+    ngx_ssl_conn_t          *ssl_conn;
+    ngx_connection_t        *c;
+    ngx_uint_t               p;
+
+    if (r->connection == NULL || r->connection->ssl == NULL) {
+        *err = "bad request";
+        return NGX_ERROR;
+    }
+
+    ssl_conn = r->connection->ssl->connection;
+    if (ssl_conn == NULL) {
+        *err = "bad ssl conn";
+        return NGX_ERROR;
+    }
+
+    c = ngx_ssl_get_connection(ssl_conn);
+
+    switch (c->local_sockaddr->sa_family) {
+        case AF_UNIX:
```

Comment:
Style: in nginx coding style, `case` should be aligned up with `switch`.

Reply:
update